lib/Connection.php

```diff
@@ -364,7 +399,7 @@ private function doSoftValidation() {
 		foreach ($saKeys as $key) {
 			$val = $this->configuration->$key;
 			if (\is_array($val) && \count($val) === 1 && empty($val[0])) {
-				$this->configuration->$key = [];
+				$this->configuration->$key = '';
```

Comment:
Comment at line 396 says:
`make sure empty search attributes are saved as simple, empty array`

A simple single empty item is now saved as an empty string.

Does the comment need to be updated?

Reply:
I am not reallly sure why it is that way. Both the properties `'ldapAttributesForUserSearch'` and `'ldapAttributesForGroupSearch'` are supposed to be strings so they should be empty strings logically.